spec/unit/copy_service_spec.rb

```diff
@@ -22,6 +22,7 @@
       # Entities::Service instance stub
       service = double('Entities::Service instance')
```

Comment:
Would using [object double](https://relishapp.com/rspec/rspec-mocks/v/3-8/docs/verifying-doubles/using-an-object-double) simplify the test, do you don't have to setup `id` for example?

Reply:
I do not think it would simplify the test. I would have to construct the service object and pass `id` and `remote` objects (doubles?) to the constructor. On the other hand it would check that any stubbed methods on the double also exist on the template, which it is not done now. 

I consider this is not a big deal for this test.

Reply:
Object double verifies that you can't stub a method that does not exist. I think [instance_double](https://relishapp.com/rspec/rspec-mocks/v/3-8/docs/verifying-doubles/using-an-instance-double) does the same and does not require the constructor.